source/docs/articles/local/drush-versions.md

```diff
@@ -38,7 +38,7 @@ function policy_drush_sitealias_alter(&$alias_record) {
   }
 }
 ```
-Commit the policy file to your site's repository and clear Drush cache by running `drush cc drush`, then check Drush version:
+Save the file and clear Drush cache by running `drush cc drush`, then check Drush version:
```

Comment:
The policy file is stored locally, usually in `~/.drush/policy.drush.inc` or similar; it is usually not committed, and when committed, it would be in some repository other than the Pantheon site repository.